src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -254,7 +254,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
                 foreach ($tagData as $parameters) {
                     $output .= "\n".'- Tag: `'.$tagName.'`';
                     foreach ($parameters as $name => $value) {
-                        $output .= "\n".'    - '.ucfirst($name).': '.$value;
+                        $output .= "\n".'    - '.ucfirst($name).': '.(\is_array($value) ? $this->formatParameter($value) : $value);
```

Comment:
We can't do a simple explode because we allow array of array of array etc. So I think it's alright to reuse the formatParameter method (see tests for example output).